tests/test_primitives/test_reawaitable/test_reawaitable_concurrency.py

```diff
@@ -0,0 +1,21 @@
+import anyio
+import pytest
+
+from returns.primitives.reawaitable import ReAwaitable
+
+
+async def sample_coro():
+    await anyio.sleep(0.1)
```

Comment:
```suggestion
    await anyio.sleep(1)
```

give less chance for random failures.